osquery/tables/networking/windows/arp_cache.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <osquery/tables.h>
+#include <string>
+
+#include "osquery/core/conversions.h"
+#include "osquery/core/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+const std::map<unsigned short, const std::string> mapOfAddressFamily = {
+    {2, "IPv4"}, {23, "IPv6"},
+};
+
+const std::map<unsigned char, const std::string> mapOfStore = {
+    {0, "Persistent"}, {1, "Active"},
+};
+
+const std::map<unsigned char, const std::string> mapOfState = {
+    {0, "Unreachable"},
+    {1, "Incomplete"},
+    {2, "Probe"},
+    {3, "Delay"},
+    {4, "Stale"},
+    {5, "Reachable"},
+    {6, "Permanent"},
+    {7, "TBD"},
+};
+
+QueryData genArpCache(QueryContext& context) {
+  QueryData results;
+
+  WmiRequest wmiSystemReq("select * from MSFT_NetNeighbor",
+                          L"ROOT\\StandardCimv2");
+  std::vector<WmiResultItem>& wmiResults = wmiSystemReq.results();
+
+  if (wmiResults.size() != 0) {
+    Row r;
```

Comment:
You could push this `Row r` declaration inside the for loop and reduce it's scope.